docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Automating Your Data Pipelines | Dagster
+description: This guide illustrates how to automate your data pipelines 
+---
+
+# Automating your data pipelines
+
+In this guide, we’ll walk you through how to run your data pipelines without manual intervention, i.e. automate, and identify the Dagster tools to make that happen.
+
+This guide assumes you have some familiarity with several Dagster concepts, including [software-defined assets,](https://docs.dagster.io/concepts/assets/software-defined-assets) [ops](https://docs.dagster.io/concepts/ops-jobs-graphs/ops) and [graphs](https://docs.dagster.io/concepts/ops-jobs-graphs/graphs), and [jobs](https://docs.dagster.io/concepts/ops-jobs-graphs/jobs), and [when to use each](https://docs.dagster.io/guides/dagster/how-assets-relate-to-ops-and-graphs).
+
+### Things to think through
+
+When thinking about automating your data pipelines, it’s helpful to think through when your data needs to be refreshed. A few helpful things to think through:
```

Comment:
If I'm understanding correctly, this guide doesn't directly tell readers how they should use the answers to these questions to inform how they should automate their data pipelines.  If that's the case, maybe we should leave them out? Otherwise we risk overwhelming them with uncertainty that we're not able to help them resolve.

Reply:
I thought it would be a good thought exercise for them to think about the holisitc picture before starting to build, removed for now..
### Things to think through

When thinking about automating your data pipelines, it’s helpful to think through when your data needs to be refreshed. A few helpful things to think through:

- How is my data split up? Do older records need updating? - partitions 
- Am I waiting for some data that needs to trigger downstream updates? - asset sensors
- Do I want it to run every time a new file is added or update data in batches? - cursor sensors 
- How predictable is my pipeline? - this we can remove since it was more on dynamic graphs 
- What are internal and external points of failure and how can i mitigate them? - remove since we excluded the failures

Reply:
I liked this section in the draft version - I thought it was really helpful. It helped me understand the types of questions I need to ask and answer before starting, and then the meat of the guide provided detail to expand things.

Reply:
added it back in and modified the questions to make sure we're covering everything